com.sayonara-player.sayonara.json

```diff
@@ -0,0 +1,149 @@
+{
+    "app-id": "com.sayonaraplayer.sayonara",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "sayonara",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
```

Comment:
do you really need host permissions?

Reply:
Since Sayonara is a media player, we do not know where the users have stored their music. I've added this because i thought that accessing /mnt/.... for smb/nfs shares needs host permissions - is there a better way?